generated/nifake/nifake/session.py

```diff
@@ -181,6 +181,11 @@ class _SessionBase(object):
 
     A property of type string with read/write access.
     '''
+    read_write_string_repeated_capability = _attributes.AttributeViStringRepeatedCapability(1000010)
+    '''Type: basic sequence types or str or int
+
+    A property of type string with read/write access, that represents a repeated capability
```

Comment:
I know this is internal, but we support and even encourage int, lists of int... etc
Look at other places where we explain it and consider using that string, or instead say something like:

> Anything nimi-python generally allows as a repeated capability (int, str, sequence of these...)